src/modules/Crop/Module.js

```diff
@@ -13,21 +13,23 @@
  *          y = options.y
  *          y = options.y + options.h
  */
- module.exports = function CropModule(options) {
+ module.exports = function CropModule(options,UI) {
    options = options || {};
    options.title = "Crop Image";
-   var this_ = this;
+   UI.onSetup();
    var output
 
    function draw(input,callback) {
 
+     UI.onDraw();
      const this_ = this;
 
      require('./Crop')(input,options,function(out,format){
        this_.output = {
```

Comment:
This doesn't error on removing `var this_ = this`?

Also, best practice is to use a meaningful name like `module` instead of this_. Thanks!